editor/src/execution/views.tsx

```diff
@@ -0,0 +1,20 @@
+import { svg } from '@eclipse-glsp/client';
+import { VNode } from 'snabbdom';
+import { Executable } from './model';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const JSX = { createElement: svg };
+
+export function createExecutionBadge(node: Executable, width: number): VNode {
+  if (node.executionCount) {
+    return (
+      <g>
+        <circle r='8' cx={width} stroke='black' stroke-width='1px' fill='white'></circle>

```

Comment:
this will look weird on a dark editor ;)
set a class to this circle and add the stroke, stroke-width and fill via css. And use the css variables for white and black